users/views.py

```diff
@@ -85,6 +93,7 @@ def get_serializer_class(self) -> Type[Union[UserUpdateByAdminSerializer, UserUp
 
 @login_required
 def index(request: HttpRequest) -> HttpResponse:
+    logger.info(f"User with id: {request.user.pk} logged in")
```

Comment:
It won't be always log in action, sometimes you could be already logged in and go to index site.

Reply:
Done